docs/reference/glossary.rst

```diff
@@ -0,0 +1,97 @@
+.. _glossary:
+
+========
+Glossary
```

Comment:
How are you expecting people to get to this? Maybe if the terms were links to here it'd be used but I don't think I'd get here.